.github/CODEOWNERS

```
@@ -238,6 +238,7 @@
 /notion/                                 @makenotion/public-api @DataDog/ecosystems-review
 /singlestoredb_cloud/                    @tdasarathan @DataDog/ecosystems-review
 /emqx/                                   @zhongwencool @zmstone @DataDog/ecosystems-review
+/anecdote/				 @atyulyutaev @DataDog/ecosystems-review
```

```suggestion
/anecdote/                                 @atyulyutaev @DataDog/ecosystems-review
```
Tabs break validations